pytest_fixtures/api_fixtures.py

```diff
@@ -29,13 +29,13 @@
 
 
 @pytest.fixture(scope='session')
-def default_lce():
-    return entities.LifecycleEnvironment().search(query={'search': 'name=Library'})[0]
+def default_org():
+    return entities.Organization().search(query={'search': 'name="Default Organization"'})[0]
```

Comment:
Please use the pre-defined variable for the default organization instead of doing hardcode.

```suggestion
    return entities.Organization().search(query={'search':  f'name="{DEFAULT_ORG}"'})[0]
```